package.json

```diff
@@ -1,113 +1,115 @@
 {
-    "name": "dropbox",
```

Comment:
Can you keep the indentation the same to preserve the git blame? Changes to this file are automated and indentation may be reversed.